AutomatedTesting/Gem/PythonTests/scripting/scripting_utils/script_canvas_component.py

```diff
@@ -65,6 +61,22 @@ def create_new_entity_with_component(self, entity_name: str, sc_file_path: str,
         self.script_canvas_component = self.editor_entity.get_components_of_type([SCRIPT_CANVAS_UI])[0]
         self.script_canvas_component.set_component_property_value(SCRIPT_CANVAS_COMPONENT_PROPERTY_PATH, sourcehandle)
 
+    def change_component_graph_file(self, sc_file_path: str):
```

Comment:
Rename to `set_componenet_graph_file`

Reply:
renamed this function. Also updated the script_canvas_component field to be a list since I looked ahead in the other tests and noticed a few that use multi-component entities.